### ISSUE TITLE
Terminal filled with warning

### ISSUE THREAD

**Issue by nicodebo** [carnager/clerk#34]
hi,
I've just switch to clerk v4 and the new interface looks great. Thanks.
I used the AUR package to install clerk.

However, when I start clerk in "album" or "sorted album" views, my terminal is filled with the following message:

```Use of uninitialized value in join or string at /usr/bin/clerk line 368, <GEN0> line 29.```

**Comment by carnager** [comment]
yeah i should filter those out. they are harmless though, it simply means you have files with missing tags.

**Comment by nicodebo** [comment]
Thank you for this helpful script. I'm slowly fixing the missing tags. I'll report back once my library is fixed but it's going to take a while.

**Comment by nicodebo** [comment]
I just want to confirm that filling the missing "albumartist", "album" and "date" tags fix my issue.
